akka-projection-cluster/src/main/scala/akka/projection/cluster/ClusterRunner.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.cluster
+
+import scala.collection.immutable
+
+import akka.actor.typed.ActorSystem
+import akka.cluster.sharding.typed.ShardedDaemonProcessSettings
+import akka.cluster.sharding.typed.scaladsl.ShardedDaemonProcess
+import akka.projection.Projection
+import akka.projection.internal.ProjectionBehavior
+
+object ClusterRunner {
+
+  def init[Envelope](
+      system: ActorSystem[_],
+      projectionName: String,
+      projections: immutable.Seq[Projection[Envelope]]): Unit =
+    init(system, projectionName, projections, ShardedDaemonProcessSettings(system))
+
+  def init[Envelope](
+      system: ActorSystem[_],
+      projectionName: String,
+      projections: immutable.Seq[Projection[Envelope]],
+      shardedDaemonSettings: ShardedDaemonProcessSettings): Unit = {
+
+    ShardedDaemonProcess(system).init[ProjectionBehavior.ProjectionCommand](
+      s"cluster-projection-runner-$projectionName",
+      projections.size,
+      i => ProjectionBehavior(() => projections(i)),
```

Comment:
I don't follow you here. A factory in which sense?

The ShardedDaemon works already with a function `Int => Behavior`

Then, because of start/stop of projection, I want the ProjectionBehavior to receive a `() => Projection` so we can restart it with new instance. 

Isn't that enough?

Reply:
but the second time `Int => Behavior` is called for same `i` then this will use the same Projection instance `projections(i)`, and that has already been stopped and can't be started again

Reply:
I think we got trapped here. The code was already updated, but not showing here. Or I haven't pushed it when we discusseed. 

The runner receives a collection of `() => Projection(..)`

So, a new one is instantiated on each restart

Reply:
ok, I thought it still was Seq[Projection]